modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,156 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+:_content-type: PROCEDURE
+[id='replacing-a-bare-metal-control-plane-node_{context}']
+= Replacing a bare metal control plane node
+
+Use the following procedure to replace an installer-provisioned {product-title} cluster.
```

Comment:
Don't we mean 'to replace a bmh'?

Reply:
Shouldn't the instructions include removing the old bmh, before adding a new one?

Reply:
This is not about replacing a whole cluster, just a single control plane node.

Reply:
Updated as suggested.